presto-main/src/main/java/io/prestosql/sql/planner/LocalDynamicFilterConsumer.java

```diff
@@ -39,37 +41,41 @@
 import static com.google.common.base.MoreObjects.toStringHelper;
 import static com.google.common.base.Verify.verify;
 import static com.google.common.collect.ImmutableMap.toImmutableMap;
+import static com.google.common.util.concurrent.MoreExecutors.directExecutor;
 import static io.prestosql.sql.DynamicFilters.Descriptor;
 import static io.prestosql.sql.DynamicFilters.extractDynamicFilters;
 import static java.util.Objects.requireNonNull;
 import static java.util.function.Function.identity;
 
-public class LocalDynamicFilter
+public class LocalDynamicFilterConsumer
```

Comment:
nit: rename should be part of separate commit

Reply:
Done